src/main/java/billgates/usecases/user_join/UserJoinUseCase.java

```diff
@@ -3,9 +3,11 @@
 import billgates.database.QueryUserData;
 import billgates.interface_adapters.DatabaseGateway;
 
+import java.util.function.ToDoubleBiFunction;
```

Comment:
I think it may not be used in your use case. Maybe you can consider to delete this one? Or you are planning to save it for future use.

Reply:
Sure!